core/src/main/java/org/mskcc/cbio/portal/servlet/CommonQueryBuilder.java

```diff
@@ -0,0 +1,620 @@
+/*
```

Comment:
This class is mostly duplicated from `QueryBuilder`. It is not easy to review and/or maintain duplicated code. We must implement the desired functionality in a better way.